options.go

```diff
@@ -51,6 +52,12 @@ func WithDecompressFn(decompressFn func(c *gin.Context)) Option {
 	}
 }
 
+func WithMinLength(minLength uint64) Option {
+	return func(o *Options) {
+		o.MinLength = minLength
+	}
```

Comment:
```suggestion
func WithMinLength(minLength uint64) Option {
	return func(o *Options) {
		o.MinLength = int(minLength)
	}
```
Or just allow an int to be passed in.

Reply:
i just changed this to use an int